aspnetcore/fundamentals/target-aspnetcore/samples/middleware.cs

```diff
@@ -0,0 +1,57 @@
+// code omitted for brevity
+
+#if NETCOREAPP3_0 || NETCOREAPP5_0
+using System.Text.Json;
+using System.Text.Json.Serialization;
+#else
+using Newtonsoft.Json;
+#endif
+
+namespace Benchmarks.Middleware
+{
+    public class JsonMiddleware
+    {
+        private static readonly PathString _path = new PathString(Scenarios.GetPath(s => s.Json));
+        private static readonly UTF8Encoding _encoding = new UTF8Encoding(false);
+        private const int _bufferSize = 27;
+#if !NETCOREAPP3_0 && !NETCOREAPP5_0
+        private static readonly JsonSerializer _json = new JsonSerializer();
```

Comment:
It seems really bizzare to me that this example is using the TFM to choose with JSON serializer to use. That might be what our benchmarking code does, but it makes for confusion when the example does something unnecessary.